src/sage/combinat/regular_sequence.py

```diff
@@ -762,6 +762,12 @@ def subsequence(self, a, b):
 
             sage: S.regenerated().subsequence(1, -4)
             2-regular sequence 0, 0, 0, 0, 1, 3, 6, 9, 12, 18, ...
+
+        Check that the zero sequence is handeled correctly (issue:`37282`)
```

Comment:
It should be "handled" instead of "handeled".

Reply:
Thank you, fixed in 6b3480e.